test/app-tap/datetime.test.lua

```diff
@@ -2819,27 +2819,27 @@ test:test("Time :set{} operations", function(test)
             'hour 6')
     test:is(tostring(ts:set{ min = 12, sec = 23 }), '2020-11-09T06:12:23+0300',
             'min 12, sec 23')
-    test:is(tostring(ts:set{ tzoffset = -8*60 }), '2020-11-09T06:12:23-0800',
+    test:is(tostring(ts:set{ tzoffset = -8*60 }), '2020-11-08T19:12:23-0800',
             'offset -0800' )
-    test:is(tostring(ts:set{ tzoffset = '+0800' }), '2020-11-09T06:12:23+0800',
+    test:is(tostring(ts:set{ tzoffset = '+0800' }), '2020-11-09T11:12:23+0800',
             'offset +0800' )
     -- timestamp 1630359071.125 is 2021-08-30T21:31:11.125Z
```

Comment:
the comment should be updated too, right?

Reply:
I suppose the timestamp is in `+0000`
I've extended the comment
```diff
@@ -2823,7 +2823,8 @@ test:test("Time :set{} operations", function(test)
             'offset -0800' )
     test:is(tostring(ts:set{ tzoffset = '+0800' }), '2020-11-09T11:12:23+0800',
             'offset +0800' )
-    -- timestamp 1630359071.125 is 2021-08-30T21:31:11.125Z
+    -- timestamp 1630359071.125 is 2021-08-30T21:31:11.125Z+0000
+    -- or 2021-08-31T05:31:11.125Z+0800
```